### ISSUE TITLE
What about Maui?

### ISSUE THREAD

**Issue by KannanKrish** [verybadcat/CSharpMath#206]
This library was supported by Maui?

**Comment by charlesroddie** [comment]
Yes this library allows can render to SkiaSharp and that supports Maui.

**Comment by syaifulnizamyahya** [comment]
Is there any simple sample on how to use CSharpMath on MAUI?

Thanks.

**Comment by syaifulnizamyahya** [comment]
Finally got it working. If anyone interested, here is an example on how to use it in MAUI.
https://github.com/syaifulnizamyahya/MauiSkiaCsharpMath

**Comment by KannanKrish** [comment]
I have the MathView component from CSharpMath.Forms. 

How can I use it in MAUI?

**Comment by KannanKrish** [comment]
I am still waiting for the maui control for csharpmath. This repo is active or not?

**Comment by charlesroddie** [comment]
The current documentation for SkiaSharp is: https://github.com/verybadcat/CSharpMath#2-csharpmathskiasharp

Some SkiaSharp documentation: https://learn.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/

> I am still waiting for the maui control for csharpmath. This repo is active or not?

The repo hasn't had commits or reviews in a reasonably long time. If you are interested in maintaining this repo, create a separate issue.
